assets/js/components/CleanUpButton/CleanUpButton.jsx

```diff
@@ -5,11 +5,13 @@ import { EOS_CLEANING_SERVICES } from 'eos-icons-react';
 import Button from '@components/Button';
 import Spinner from '@components/Spinner';
 
-function CleanUpButton({ cleaning, onClick }) {
+function CleanUpButton({ cleaning, onClick, className = '' }) {
```

Comment:
You don't need to default to empty string I think.
I would work without default as well